src/renderer/components/ft-input-tags/ft-input-tags.css

```diff
@@ -38,6 +38,12 @@
   user-select: text;
 }
 
+.checkbox-container {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+}
```

Comment:
```suggestion
.checkbox-container {
  display: flex;
  align-items: center;
  font-size: 12px;
  accent-color: var(--accent-color);
}
```

**suggestion:** Use app's accent color (the secondary theme color) for checkbox checked color for improved visual consistency.